packages/react/src/CTABanner/CTABanner.tsx

```diff
@@ -13,58 +13,130 @@ import '@primer/brand-primitives/lib/design-tokens/css/tokens/functional/compone
 /** * Main Stylesheet (as a CSS Module) */
 import styles from './CTABanner.module.css'
 
+export const CTABannerBackgroundColors = ['default', 'subtle'] as const
+
+type ResponsiveMap<T> = {
+  narrow?: T
+  regular?: T
+  wide?: T
+}
+
+type ResponsiveBackgroundImageSrcMap = ResponsiveMap<string | string[]>
+type BackgroundColors = (typeof CTABannerBackgroundColors)[number] | AnyString
+type ResponsiveBackgroundColorMap = ResponsiveMap<BackgroundColors>
+type ResponsiveBackgroundImagePositionMap = ResponsiveMap<string | string[]>
+type ResponsiveBackgroundImageSizeMap = ResponsiveMap<string | string[]>
+
 export type CTABannerProps = BaseProps<HTMLElement> &
   React.HTMLAttributes<HTMLElement> & {
-    children: React.ReactNode | React.ReactNode[]
+    /**
+     * The alignment of the content within the banner.
+     */
     align?: 'start' | 'center'
+    /**
+     * A flag to add a border to the banner.
+     */
     hasBorder?: boolean
+    /**
+     * A flag to remove the shadow from the banner.
+     */
     hasShadow?: boolean
+    /**
+     * A flag to remove the background from the banner.
+     */
     hasBackground?: boolean
+    /**
+     * Optional, custom background color
+     */
+    backgroundColor?: BackgroundColors | ResponsiveBackgroundColorMap
+    /**
+     * Optional, custom background image
+     */
+    backgroundImageSrc?: string | ResponsiveBackgroundImageSrcMap
+    /**
+     * Optional, custom background position
+     */
+    backgroundImagePosition?: string | ResponsiveBackgroundImagePositionMap
+    /**
+     * Optional, custom background size
+     */
```

Comment:
```suggestion
    /**
     * Optional, custom background color.
     */
    backgroundColor?: BackgroundColors | ResponsiveBackgroundColorMap
    /**
     * Optional, custom background image.
     */
    backgroundImageSrc?: string | ResponsiveBackgroundImageSrcMap
    /**
     * Optional, custom background position.
     */
    backgroundImagePosition?: string | ResponsiveBackgroundImagePositionMap
    /**
     * Optional, custom background size.
     */
```

Nitpick, for consistency.